types/trigger.d.ts

```diff
@@ -26,12 +26,12 @@ export type LocaleText = LocaleObject<string>;
 
 export type ZoneId = number | null;
 
+export type OutputStrings = { [outputKey: string]: LocaleText };
```

Comment:
```suggestion
export type OutputStrings = { [outputKey: string]: LocaleText | string };
```

I think this is what you're describe that the trigger does.

Reply:
I wasn't sure if that was applicable to both cases (the `outputStrings` trigger property vs the `responseOutputStrings` in-trigger-response override).

Frankly, I think it might make more sense to enforce `LocaleText` on the value here, but as I said before I'm not sure if there's a good way to find other cases that aren't using the correct structure.